types/prompts/index.d.ts

```diff
@@ -78,7 +78,7 @@ declare namespace prompts {
         type: PromptType | Falsy | PrevCaller<T, PromptType | Falsy>;
         name: ValueOrFunc<T>;
         message?: ValueOrFunc<string>;
-        initial?: string | number | boolean | Date;
+        initial?: ValueOrFunc<string>;
```

Comment:
[Numbers](https://github.com/terkelg/prompts#numbermessage-initial-max-min-style), [booleans](https://github.com/terkelg/prompts#togglemessage-initial-active-inactive), and [dates](https://github.com/terkelg/prompts#datemessage-initial-warn) are also valid values here as noted in the docs of other prompts functions

Reply:
Fixed.